examples/pimlico/src/mint.js

```diff
@@ -46,6 +44,8 @@ const txHash = await smartAccountClient.sendTransaction({
     to: contractAddress,
     data: callData,
     value: BigInt(0),
+    maxFeePerGas: 3000000000n,
+    maxPriorityFeePerGas: 1000000000n,
```

Comment:
Temporary - permissionless expects the bundler to share the same url as the other jsonrpc requests. Needed to hardcode these values here because `eth_getBlock` was failing.

This won't be necessary once rundler is available through RPC proxy, as they will all share the same URL